webapp/components/survey/NodeDefDetails/hooks/useMultipleEditDisabled.js

```diff
@@ -0,0 +1,20 @@
+import * as Survey from '@core/survey/survey'
+import * as NodeDef from '@core/survey/nodeDef'
+import * as NodeDefLayout from '@core/survey/nodeDefLayout'
+
+import { useSurvey, useSurveyCycleKey } from '@webapp/store/survey'
+
+export const useIsMultipleEditDisabled = (nodeDef) => {
```

Comment:
arguments must be passed in object. e.g {nodeDef}

Reply:
ok